README.md

```diff
@@ -1,14 +1,19 @@
-# CABLE benchmarking
+# benchcab
 
-[![codecov](https://codecov.io/gh/CABLE-LSM/benchcab/branch/master/graph/badge.svg?token=JJYE1YZDXQ)](https://codecov.io/gh/CABLE-LSM/benchcab)
+[![Documentation status][readthedocs_badge]][docs] [![Test coverage][codecov_badge]][codecov_summary] [![Conda package status][conda_badge]][conda]
 
-Repository to benchmark CABLE. The benchmark will run the exact same configurations on two CABLE branches specified by the user, e.g. a user branch (with personal changes) against the head of the trunk. The results should be attached with all new [tickets](https://trac.nci.org.au/trac/cable/report/1).
+`benchcab` is a testing framework that tests the CABLE land surface model across a range of model configurations and model versions. The tool checks out the model versions specified by the user, builds the required executables, and runs each model version across N standard science configurations. Model outputs are then piped into a benchmark analysis via [modelevaluation.org][meorg] from which the user can assess model performance.
```

Comment:
```suggestion
`benchcab` is a testing framework that tests the CABLE land surface model across a range of model configurations and model versions. The tool:

- checks out the model versions specified by the user
- builds the required executables
- runs each model version across N standard science configurations
- performs a bitwise comparison between CABLE simulations for the same science configuration but different model versions. 

The user can then pipe the model outputs into a benchmark analysis via [modelevaluation.org][meorg] to assess model performance.

```
The steps are clearer in a list because of the number of them.

I want to make obvious the last step is manual. I know it will have to be changed but I don't know when the API will come in.